pyuvsim/simsetup.py

```diff
@@ -691,17 +757,19 @@ def parse_time_params(time_params):
 
     Args:
         time_params: Dictionary of time parameters
+            See pyuvsim documentation for examples of allowable key combinations.
```

Comment:
same comment

Reply:
I'll add the url to the readthedocs page/section.